src/pytorch_lightning/trainer/trainer.py

```diff
@@ -195,111 +193,87 @@ def __init__(
                 trying to optimize initial learning for faster convergence. trainer.tune() method will
                 set the suggested learning rate in self.lr or self.learning_rate in the LightningModule.
                 To use a different key set a string instead of True with the key name.
-                Default: ``False``.
```

Comment:
Some of these were added by contributors: #11614

As a general guideline I agree, but it also doesn't hurt having them in something like the core Trainer API (that doesn't really change their defaults that often). 

I propose to keep these removals to a minimum, and to keep them in the Trainer.

Reply:
I see, not sure why we moved some back...
In my point of view, I am very fine to keep the default when there is some additional explanation or referring to the class type or just in text after a value set that the one is default... but having pure sentence "Default: None." I think is useless...

Reply:
I suggest keeping the "default:" message for the arguments that are Optional in the code only so that we can know whether the user manually passed an argument or not to then set the real default. All others could be removed

Reply:
> arguments that are Optional in the code only so that we can know whether the user manually passed an argument

not sure if I understand what you mean here :confused:

Reply:
We often use the pattern

```py
x: Optional[int]
if x is None:
    # the user didn't set it
    self.x = 10
else:
    self.x = x
```
instead of

```pyt
x: int = 10
self.x = x
```

So for those, we want to still mention `Default: 10` in the docstring because saying `Default: None` is not informative unless it also explains that `None` gets converted into `10`

Reply:
yes agree, but so far it was not such and we just said default None...

Reply:
I'm ok with adjusting the other less important files. But my opinion remains unchanged regarding trainer.py, I'd like that it stays and that we don't go back and forth #11614

Reply:
But @Borda how about we merge the PR but remove the changes from Trainer. That's what I'm trying to say.